kolibri/plugins/learn/assets/src/views/CompletionModal/index.vue

```diff
@@ -472,4 +508,9 @@
     }
   }
 
+  .loader {
+    margin-top: 16px;
+    margin-bottom: 16px;
```

Comment:
A minor note - are there designs for the loading state? I thought it could be nice to have more space below the loader. Right now it seems to be very close to the bottom border to me:
![Screenshot from 2021-11-29 09-54-27](https://user-images.githubusercontent.com/13509191/143851414-8266b90b-4b43-40fa-bd08-64fa66c3e064.png)

cc @jtamiace

Reply:
There was not a design for this - which I think was an oversight, happy to get some additional direction on its precise look and feel.

Reply:
Circular loader is the right component for this. A little more space would be nice - 56px both above and below?

Reply:
![Screenshot from 2021-11-29 11-29-05](https://user-images.githubusercontent.com/1680573/143930437-c8db7aa1-3b24-499f-9fee-52276c287dd3.png)